packages/linux/package.mk

```diff
@@ -308,4 +308,14 @@ post_install() {
     if grep -q ^CONFIG_CFG80211_REQUIRE_SIGNED_REGDB= $PKG_BUILD/.config; then
       cp $(get_build_dir wireless-regdb)/regulatory.db{,.p7s} $INSTALL/$(get_full_firmware_dir)
     fi
+  # modprobe config files for project/device
+    if [ -d $PROJECT_DIR/$PROJECT/devices/$DEVICE/linux ]; then
+      modprobe_config_dir="$PROJECT_DIR/$PROJECT/devices/$DEVICE/linux/modprobe.d"
+    else
+      modprobe_config_dir="$PROJECT_DIR/$PROJECT/linux/modprobe.d"
+    fi
+    if [ -d "$modprobe_config_dir" ]; then
+      mkdir -p $INSTALL/usr/lib/modprobe.d
+      cp $modprobe_config_dir/*.conf $INSTALL/usr/lib/modprobe.d
+    fi
```

Comment:
Edit: quick correction to my previous post - the ath9k.conf paths should be `packages` not `package` (previous post updated).

Reply:
Thinking about this some more...  a better solution might be to build on https://github.com/LibreELEC/LibreELEC.tv/blob/master/scripts/install#L113-L116 where we already install modprobe.d (but only for the global package, and not for projects or devices).

I wonder if we should extend the logic in `scripts/install` so that it iterates over `$PKG_DIR`, `$PROJECT` and `$DEVICE` installing the conf files as necessary. I also wonder if we should do that for everything in the `if` block - ie. debug.d, sysconf.d etc. Maybe something for another PR...

Reply:
Can you consider cherry-picking this commit: https://github.com/LibreELEC/LibreELEC.tv/compare/master...MilhouseVH:le90_fix_install_proj_dev (it's totally untested, btw)

It should make any modifications to linux/package.mk unnecessary for this PR - just install the two ath9k.conf files as I previously mentioned, for the Generic and RPi projects.